ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -90,6 +90,8 @@ def test_numpy_nanmean(
         test_flags=test_flags,
         fn_tree=fn_tree,
         on_device=on_device,
+        rtol=1e-1,
```

Comment:
We should always avoid passing in such a large tolerance for functions like these which do not do high computations, try reverting both to `1e-2` at maximum, not more than that.